aws/src/integration/java/org/apache/iceberg/aws/s3/TestS3FileIOIntegration.java

```diff
@@ -419,7 +436,7 @@ public void testDeleteFilesSingleBatchWithRemainder() throws Exception {
   public void testPrefixList() {
     S3FileIO s3FileIO = new S3FileIO(clientFactory::s3);
     List<Integer> scaleSizes = Lists.newArrayList(1, 1000, 2500);
-    String listPrefix = String.format("s3://%s/%s/%s", bucketName, prefix, "prefix-list-test");
+    String listPrefix = String.format("s3://%s/%s/%s", bucketName, "prefix-list-test", prefix);
```

Comment:
wasn't 100% sure whether to swap these around or just a "/" to the "prefix-list-test" string. Open to change

Reply:
I'd probably keep them the same order as previously, and either add a `/` to the end of the original string or as you suggested